src/main/webapp/WEB-INF/views/jsp/home.jsp

```diff
@@ -20,8 +20,11 @@
             <c:choose>
                 <c:when test="${empty cookie.name.value}">
                     <div class="startButtons">
+                        <!--
+                        Commented SIGNIN and REGISTER buttons until their implementation appears
                         <a class="signInButton increaseDistance" href="${pageContext.request.contextPath}/login">Sign In</a>
                         <a class="signInButton" href="${pageContext.request.contextPath}/registration">Register</a>
+                        -->
```

Comment:
I'm not sure if the registration and login buttons are only on the home page. Could you check it please?
You need to make sure that they are not on other pages.